cloud-build/cloudbuild.yaml

```diff
@@ -0,0 +1,7 @@
+steps:
+# Create docker image from regular Dockerfile
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/google.com/hadoop-cloud-dev/init-actions-image', '-f', 'cloud-build/Dockerfile', '.']
+- name: 'gcr.io/google.com/cloud-dataproc-ci/init-actions-image'
```

Comment:
In the previous step, the project is hadoop-cloud-dev, but here it is cloud-dataproc-ci, why?

Reply:
Great catch! 

Wow, I spend hours yesterday debugging why the packages installed in the Dockerfile weren't accessible in the script. This seems to be it, thanks!